src/components/Application/Navigation.tsx

```diff
@@ -0,0 +1,133 @@
+import React from 'react'
+import { Button, Segment, Sticky } from 'semantic-ui-react'
+import {
+  MethodRevalidate,
+  MethodToCallProps,
+  SectionAndPage,
+  SectionDetails,
+  SectionsStructureNEW,
+} from '../../utils/types'
+import strings from '../../utils/constants'
+import { useRouter } from '../../utils/hooks/useRouter'
+
+interface NavigationProps {
+  current: SectionAndPage
+  sections: SectionsStructureNEW
+  serialNumber: string
+  requestRevalidation: MethodRevalidate
+}
+
+const Navigation: React.FC<NavigationProps> = ({
+  current,
+  sections,
+  serialNumber,
+  requestRevalidation,
+}) => {
+  const { push } = useRouter()
+
+  const currentSectionDetails = sections[current.sectionCode].details
+
+  const nextSection = (): SectionDetails | null => {
```

Comment:
hmm wonder why these need to be a method ?

Reply:
Just to check before accessing the first one from the returned array (of next sections). Do you think it could be done in one line?

Reply:
could just be 
```JS
const nextSections = Object.values(sections)
      .filter(({ details: { index } }) => index > currentSectionDetails.index)
      .sort(({ details: { index: aIndex } }, { details: { index: bIndex } }) => aIndex - bIndex)
const nextSection = nextSections[0].details
      
```

Reply:
Hmmm ok, there is also the check for null required there. But I'll do this change, it would be better to not have methods indeed. Thanks

Reply:
Done: 4bd56a7